src/views/Home/Home.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import Controls from '../../components/Controls/Controls';
+import PokeList from '../../components/PokeList/PokeList';
+import { getPokemon } from '../../services/pokemon';
+import './Home.css';
+
+export default function Home() {
+  const [loading, setLoading] = useState(true);
+  const [pokemon, setPokemon] = useState([]);
+  const [query, setQuery] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await getPokemon(query);
+      setPokemon(data.results);
+      setLoading(false);
+    };
+    fetchData();
+  }, [query]);
+
+  const handleSubmit = (e) => {
```

Comment:
I might only re-fetch the pokemon in this handleSubmit function so they are not fetched on each input